driver-core/src/main/com/mongodb/internal/operation/MixedBulkWriteOperation.java

```diff
@@ -262,17 +241,17 @@ public void executeAsync(final AsyncWriteBinding binding, final SingleResultCall
                     return;
                 }
                 logRetryExecute(retryState);
-                executeBulkWriteBatchAsync(retryState, binding, connection, maxWireVersion, releasingCallback);
+                executeBulkWriteBatchAsync(retryState, binding, connection, releasingCallback);
             });
         }).whenComplete(binding::release);
         retryingBulkWrite.get(exceptionTransformingCallback(errorHandlingCallback(callback, LOGGER)));
     }
 
-    private BulkWriteResult executeBulkWriteBatch(final RetryState retryState, final WriteBinding binding, final Connection connection,
-            final int maxWireVersion) {
+    private BulkWriteResult executeBulkWriteBatch(final RetryState retryState, final WriteBinding binding, final Connection connection) {
         BulkWriteTracker currentBulkWriteTracker = retryState.attachment(AttachmentKeys.bulkWriteTracker())
                 .orElseThrow(Assertions::fail);
         BulkWriteBatch currentBatch = currentBulkWriteTracker.batch().orElseThrow(Assertions::fail);
+        int maxWireVersion = connection.getDescription().getMaxWireVersion();
```

Comment:
Removed the redundancy: there is no need to pass `maxWireVersion` when we can get it from `connection`.

Reply:
Ack